docs/developer/docker-compose.md

```diff
@@ -0,0 +1,75 @@
+# Local Development with Docker Compose
+
+Developers can use Docker Compose to get a local development environment
+
+## Running
+
+```
+$ docker-compose -f docker-compose.dev.yml up
+```
+
+The website should be available at http://localhost:8000
+
+Use Ctrl-C to exit.
+
+Note you will get an "index_not_found_exception" until you load some data.
+
+## Loading Basic Data
+
+While you do this, the Elasticsearch server must be running - use the general up command.
+
+To load some basic data:
+
+```
+$ docker-compose -f docker-compose.dev.yml run grantnav-web su -c 'wget -O  scvo-test-data.json https://scvo.scot/support/coronavirus/funding/scottish-government/community-recovery/crf/recipients/data.json && python dataload/import_to_elasticsearch.py --clean scvo-test-data.json'
+```
+
+## Loading Full Data
+
+To do this you will need to know the URL, username and password to a [DataStore server](https://github.com/ThreeSixtyGiving/datastore).
+
+(If you work for 360, the URL is `https://store.data.threesixtygiving.org/api/grantnav/updates` and the username and password can be found in [the private deploy repo](https://github.com/ThreeSixtyGiving/threesixtygiving-deploy-pillar-private).)
```

Comment:
Prefer if you didn't say this, it's a 360 policy thing, some people do have access outside of 360 to the datastore.